boxoffice/models/item.py

```diff
@@ -26,6 +33,19 @@ class Item(BaseScopedNameMixin, db.Model):
 
     parent = db.synonym('item_collection')
 
+    gst_type = db.Column(db.SmallInteger, default=GST_TYPE.SERVICE, nullable=True)
+    # Harmonized System of Nomenclature code
+    hsn = db.Column(db.Unicode(255), nullable=True)
+    # Services Accounting Code
+    sac = db.Column(db.Unicode(255), nullable=True)
+    cgst_tax_rate = db.Column(db.SmallInteger, nullable=True, default=0)
+    sgst_tax_rate = db.Column(db.SmallInteger, nullable=True, default=0)
+    igst_tax_rate = db.Column(db.SmallInteger, nullable=True, default=0)
+    gst_compensation_cess = db.Column(db.SmallInteger, nullable=True, default=0)
+    state_code = db.Column(db.Unicode(4), nullable=True)
+    # ISO country code
+    country_code = db.Column(db.Unicode(2), nullable=True)
```

Comment:
We're storing all of these fields on the `Item` model? What happens in the following scenario?

1. We're selling an item (at multiple price points) from July 1 to November 30 for an event on December 1.
2. On July 31, the government announces that the GST rate for event tickets will change from 18% back to 15% effective September 1.
3. We have the month of August to implement a rate change starting from September. Where is this change recorded in our software? If you edit the Item record's rates, it starts applying immediately. Do you set a reminder to edit it on the midnight of September 1?